src/runner/mod.rs

```diff
@@ -193,9 +193,10 @@ impl Runner {
             (true, true) => panic!("not currently supported"),
             (true, false) => {
                 let cfg = specular::sequencing::config::Config::new(&self.config);
-                let policy = specular::sequencing::AttributesBuilder::new(cfg);
-                let provider = Provider::try_from(self.config.l1_rpc_url.clone())?;
-                let sequencing_src = sequencing::Source::new(policy, provider);
+                let l2_provider = Provider::try_from(&self.config.l2_rpc_url)?;
+                let policy = specular::sequencing::AttributesBuilder::new(cfg, l2_provider);
```

Comment:
Why not import?

Reply:
I'm following ujval's code. I guess it is because `specular::sequencing` conflicts with `sequencing` and also `Config` is used so many times. But I can do import and rename here. cc @ujvl